custom/icds_reports/tests/__init__.py

```diff
@@ -94,6 +94,10 @@ def setUpModule():
         adapters = [get_indicator_adapter(config) for config in configs]
 
         for adapter in adapters:
+            try:
+                adapter.drop_table()
```

Comment:
I think there's a sqlalchemy.Table param (`check_first`?) to check if the table exists first before doing the operation that could take the place of this try,except block

Reply:
This isn't directly sql alchemy, instead it's from our sql adapter. we do pass check first there, but they don't have a way to cascade drops which we need whenever we use table partitioning